doc/developer.md

```diff
@@ -76,6 +76,14 @@ To update rust to the latest version, use `rustup update`.
 
 To update your Python dependencies to the latest required for sourmash, you can run `pip install -r requirements.txt`.
 
+## Installing Sourmash in the editable mode
+
+In order to use Sourmash in the command line interface or the API while developing, you will need to install it in your enviornment in editable mode by running
+
+```
+pip install -e .
```

Comment:
```suggestion
pip install -e ".[test]"
```

ref https://github.com/dib-lab/sourmash/issues/1482

Reply:
actually, I'm not certain if you can do this with just this command, or if you need both...

Reply:
@luizirber can you please post your recommended set of commands using tox?

Reply:
I use `tox -e docs` for building the Python docs, and I don't use editable mode. If needed, then I use
```tox -e dev```
and follow the instructions for activating the new environment (usually
```source .tox/dev/bin/activate```
but can be 
```conda activate .tox/dev```
for people using `tox-conda`)